HelloWorld/test.java

```diff
@@ -0,0 +1,45 @@
+package com.ej.alumni.admin.zendesk.model;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "test.java" should be located in "com/ej/alumni/admin/zendesk/model" directory, not in "/storage/workspace/alumni gitsonar test/HelloWorld". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.alumni-services-001.com/coding_rules#rule_key=squid%3AS1598)